### PR TITLE
Bearnades and spawn grenade descriptions

### PR DESCRIPTION
**Pull request by SonixApache**
Adds bear grenades for bus.
bearnades
https://streamable.com/hqky0
Also adds / fixes the descriptions for the rest of the grenades.

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/grenade/spawnergrenade
-	desc = "It is set to detonate in 5 seconds. It will unleash unleash an unspecified anomaly into the vicinity."
+	desc = "Will unleash unleash an unspecified anomaly into the vicinity when triggered."
```
Comment (boy2mantwicethefam) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm]: unleash unleash

**Comment by utilitas-** [comment]
Now just add a bearnade clusterbang.

**Comment by SonixApache** [comment]
clusterbears are a distant future, would have to refactor IEDs and clusterbangs down to base grenade level

**Comment by utilitas-** [comment]
But _**clusterbears.**_

**Pull request merged**